doc/en/user/source/services/wms/vendor.rst

```diff
@@ -357,3 +357,17 @@ An example with XML POST:
                xmlns:gml="http://www.opengis.net/gml"
       version="1.1.1" service="WMS" rootLayer="false">
    </ogc:GetCapabilities>
+
+   
+clip
+------
+
+The ``clip`` parameter can be used to clip WMS response using a Polygon mask represented by a valid WKT String.
```

Comment:
@aaime during WKT parsing, if no SRID is defined, then the SRS from map request will be assumed. If an SRID is defined, it will re-project the mask geometry into map request SRID, because it will be required to perform optimization (e.g mask completely inside /outside bbox).